subprojects/common/graphite/src/main/kotlin/com/avito/android/graphite/Graphite.kt

```diff
@@ -0,0 +1,18 @@
+package com.avito.android.graphite
+
+/**
+ * https://graphite.readthedocs.io/en/latest/terminology.html#term-value
+ */
+fun graphiteSeriesElement(value: String) = value.replace(invalidSymbols, "_")
+
+/**
+ * https://graphite.readthedocs.io/en/latest/terminology.html#term-series
+ */
+fun graphiteSeries(vararg element: String) = element.joinToString(
```

Comment:
Advice: add validation for empty element

Reply:
Removed these functions. It was a copy from statsd module